surfsense_backend/app/tasks/connectors_indexing_tasks.py

```
@@ -11,8 +11,11 @@
 from app.connectors.notion_history import NotionHistoryConnector
 from app.connectors.github_connector import GitHubConnector
 from app.connectors.linear_connector import LinearConnector
+from app.connectors.discord_connector import DiscordConnector
+from discord import DiscordException
```

Ruff (0.11.9)

`discord.DiscordException` imported but unused

Remove unused import: `discord.DiscordException`

(F401)